oppia/views.py

```diff
@@ -276,8 +276,8 @@ def upload_step2(request, course_id, editing=False):
                                'editing': editing,
                                'title': page_title})
 
-def add_course_tags(form, course):
-    tags = form.cleaned_data.get("tags").strip().split(",")
+def add_course_tags(form, course, user):
+    tags = form.cleaned_data.get("tags", "").strip().split(",")
```

Comment:
Also updated this line because the `dict.get` method returns `None` if no other default is provided if the key is not present, and that would result in an `AttributeError`.